config/samples/gatling-operator_v1alpha1_gatling01.yaml

```diff
@@ -42,12 +47,39 @@ spec:
     #      secretKeyRef:
     #        name: aws-credentail-secrets
     #        key: AWS_SECRET_ACCESS_KEY
+    #
+    #############################################################
+    #        Storage Provider - gcp (Google Cloud Storage)
+    #############################################################
+    # provider: "gcp"
+    # bucket: "gatling-operator-reports"                  # GCS bucket name on which Gatlilng report files are stored
+    # bucket: "zozo-mlops-dev-gatling-operator-reports"   # レポート用のGCSバケット zozo-mlops-dev(stg)-gatling-operator-reports を定義しているのでそちらを指定してください
+    #
+    #############################################################
+    # .      Storage Provider - azure (Azure Blob Storage) ########
+    #############################################################
+    #provider: "azure"
+    #bucket: "gatling-operator-reports"                   # Azure Blob Storage container name on which Gatlilng report files are stored
+    #env:                                                 # Optional. Environment variables to be used for connecting to the cloud providers
+    #  - name: AZUREBLOB_ACCOUNT                          # Azure Blob Storage Account Name
+    #    value: xxxxxxxxxxxxxxx
+    #  - name: AZUREBLOB_KEY                              # Azure Blob Access Key. Leave blank to use SAS URL
+    #    valueFrom:
+    #      secretKeyRef:
+    #        name: azure-credentail-secrets
+    #        key: AZUREBLOBSS_KEY
+    #  - name: AZUREBLOB_SAS_URL                          # SAS URL. "Read, Write, List" permissions are required
+    #    valueFrom:
+    #      secretKeyRef:
+    #        name: azure-credentail-secrets
+    #        key: AZUREBLOB_SAS_URL
   notificationServiceSpec:
     provider: "slack"                                     # Notification provider name. Supported provider: "slack"
     secretName: "gatling-notification-slack-secrets"      # The name of secret in which all key/value sets needed for the notification are stored
   testScenarioSpec:
     # startTime: 2021-09-10 08:45:31                      # Optional. Start time of running test scenario in UTC. Format: %Y-%m-%d %H:%M:%S
-    parallelism: 3                                        # Optional. Default: 1. Number of pods running at any instant
+    parallelism: 1                                        # Optional. Default: 1. Number of pods running at any instant
+    #parallelism: 3                                        # Optional. Default: 1. Number of pods running at any instant
```

Comment:
In this sample, I think `parallelism` is intended to be `3`, is it not?

Reply:
you're light. it was just for testing https://github.com/st-tech/gatling-operator/pull/59/commits/c49c5091eca5c3ca5a0f5e77d17667e2c98acf21